movieApp/movieApp/Utility/Extensions/UIFontExtension.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  UIFontExtension.swift
+//  movieApp
+//
+//  Created by Can Kurtur on 14.08.2021.
+//
+
+import UIKit
+
+extension UIFont {
+    static let headline1 = getSFText(.bold, size: 34)
+    static let headline2 = getSFText(.bold, size: 22)
+    static let headline3 = getSFText(.bold, size: 20)
+    static let body1 = getSFText(.regular, size: 15)
+    static let body2 = getSFText(.regular, size: 12)
+    static let point = getSFText(.black, size: 10)
+
+    public enum OpenSFProType: String {
```

Comment:
I think we should not access these enum or method under UIFont type so we should separate these function to their related types.